models/Asset/Thumbnail/ImageThumbnailTrait.php

```diff
@@ -429,4 +430,20 @@ private function getFilename(): string
 
         return basename($pathReference['src']);
     }
+
+
+    /**
+     * @description Returns path for thumbnail image in a given file format
+     * @param string $format
+     * @return self
+     */
```

Comment:
Are you sure you don't want to change this?

The `@description` tag isn't used anywhere else in Pimcore and I think the `@return $this` is more correct if you `return $this;`.

```suggestion
    /**
     * Returns path for thumbnail image in a given file format
     *
     * @param string $format
     * @return $this
     */
```